Tests/WPFTests/Data/TestData.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.ObjectModel;
+using MailSender.Lib.Entities;
+
+namespace MailSender.Lib.Data
+{
+    public static class TestData
+    {
+        public static readonly ObservableCollection<Server> Servers = new ObservableCollection<Server>
```

Comment:
Не используем поля!!!

```suggestion
        public static readonly ObservableCollection<Server> Servers { get; } = new ObservableCollection<Server>
```